charts/k8tz/values.yaml

```diff
@@ -13,6 +13,9 @@ verbose: false
 # Labels to apply to all resources
 labels: {}
 
+skipInjectNamespaces:
```

Comment:
I think we need to remove `review.Request.Namespace != metav1.NamespaceSystem` in https://github.com/k8tz/k8tz/blob/master/pkg/admission/admission.go#L147-L148, then it's able to inject kube-system namespace.

Reply:
The change is more about being able to skip certain namespaces from having k8tz injected to. In certain cases (e.g. the one which I have mentioned in #76) it really makes sense to be able to configure this. And I understand that you already have an annotation one can add to a namespace, but I prefer not to have to change the kube-system namespace to achieve this behavior.

If you want to change the default behavior to also inject to the kube-system namespace (which could possible be a breaking change, since at the moment those pods are not affected by k8tz), I can change the default in this line to be `[]`.

Reply:
Ok, I understand.

Reply:
I pushed change to remove this condition.

Reply:
👍  need to update the PR title as well